src/polyfem/assembler/PressureAssembler.hpp

```diff
@@ -0,0 +1,191 @@
+#pragma once
+
+#include <polyfem/assembler/Assembler.hpp>
+#include <polyfem/mesh/Obstacle.hpp>
+
+#include <polyfem/assembler/Problem.hpp>
+#include <polyfem/assembler/MatParams.hpp>
+#include <polyfem/mesh/LocalBoundary.hpp>
+
+namespace polyfem
+{
+	namespace assembler
+	{
+		class ThermodynamicProcess
+		{
+		public:
+			ThermodynamicProcess() {}
+			virtual ~ThermodynamicProcess(){};
+			virtual double pressure(const double start_pressure, const double v0, const double v1) const { return 0; }
+			virtual double first_derivative(const double start_pressure, const double v0, const double v1) const { return 0; }
+			virtual double second_derivative(const double start_pressure, const double v0, const double v1) const { return 0; }
+			virtual double energy(const double start_pressure, const double v0, const double v1) const { return 0; }
+
+		protected:
+			const double atmospheric = 101.3e3;
```

Comment:
this should be parameters?

Reply:
I don't think so, this is pretty much constant at sea level on earth. We can rethink if there's very special circumstances to use something else.